tf2_ros/include/tf2_ros/static_transform_broadcaster.h

```diff
@@ -1,10 +1,10 @@
 /*
  * Copyright (c) 2008, Willow Garage, Inc.
  * All rights reserved.
- * 
+ *
```

Comment:
@tfoote this commit has some unrelated whitespace removal. Ok to merge with it?

Reply:
I'd rather not strip the whitespace so we can stay as compatible as possible with the ROS1 repos for merging features across.